### DISCUSSION TITLE
RNodeconf T-Echo

### DISCUSSION THREAD

**Discussion by githubioiguru** [markqvist/Reticulum#584] [Help & Questions]
Very good day,
What is the current status of the support for T-echo in Rnodeconf? Seems like fw is not available on unsigned.io?
---------------------------------------------------------------------------
                     LilyGo T-Echo RNode Installer
Serial port     : COM3
Device type     : LilyGO T-Echo 430 - 510 Mhz
Platform        : NRF52
Device MCU      : Nordic Semiconductor nRF52840
Firmware file   : rnode_firmware_techo.zip

------------------------------------------------------------------------------

Is the above correct? [y/N] Y
[12:18:20] Checking firmware file availability...
[12:18:20] Retrieving latest version info from https://unsigned.io/firmware/latest/?v=2.2.0&variant=rnode_firmware_techo.zip
[12:18:23] No valid version found for this board, exiting.

**Comment by Vertikar** [comment]
> Very good day,
> What is the current status of the support for T-echo in Rnodeconf? Seems like fw is not available on unsigned.io?
> ---------------------------------------------------------------------------
>                      LilyGo T-Echo RNode Installer
> Serial port     : COM3
> Device type     : LilyGO T-Echo 430 - 510 Mhz
> Platform        : NRF52
> Device MCU      : Nordic Semiconductor nRF52840
> Firmware file   : rnode_firmware_techo.zip
> 
> ------------------------------------------------------------------------------
> 
> Is the above correct? [y/N] Y
> [12:18:20] Checking firmware file availability...
> [12:18:20] Retrieving latest version info from https://unsigned.io/firmware/latest/?v=2.2.0&variant=rnode_firmware_techo.zip
> [12:18:23] No valid version found for this board, exiting.

Seems to be working now, I just did a fresh install and have it up and running on my t-echo
